src/components/freelancerJobs/freelancerPage.tsx

```diff
@@ -72,12 +81,25 @@ const FreelancerPage: FC = () => {
 		));
 	}, [skillsOptions]);
 
+	const filteredSkills = useMemo(() => skillsOptions.filter(s => s.value), [skillsOptions]);
+	const userSkills = useMemo(() => filteredSkills.map(s => s.name), [filteredSkills]);
+
+	if (isLoading) {
+		return <p>Loading...</p>;
+	}
+
+	const category = { value: userInfo.jobCategory.name, label: userInfo.jobCategory.name };
+	const skills = skillsMock.map(skill => ({
+		...skill,
+		value: userInfo.jobSkills.some((jobSkill: { name: string }) => jobSkill.name === skill.name),
+	}));
```

Comment:
no, I said it in the call and repeat again. in component you should have only hooks, handlers and render, that's all

Reply:
fixed